<!DOCTYPE html>
<html class="">
<head>
  <meta charset="UTF-8">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="description" content="Samson Koelle">
  <meta rel="author" href="https://plus.google.com/u/0/#">
  <meta rel="publisher" href="https://plus.google.com/u/0/#">

  <title>
    
      Contact | Samson Koelle
    
  </title>

  <link rel="stylesheet" href="/assets/css/all.css">
  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
  <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
  <!--[if lt IE 9]><script src="http://html5shim.googlecode.com/svn/trunk/html5.js"></script><![endif]-->

  <!-- Use Atom -->
  <link type="application/atom+xml" rel="alternate" href="http://localhost:4000/feed.xml" title="Samson Koelle" />

  <script type="text/javascript" async
    src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_CHTML">
  </script>

  
<script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

  ga('create', '', 'auto');
  ga('send', 'pageview');
</script>



  <!-- Use Jekyll SEO plugin -->
  <!-- Begin Jekyll SEO tag v2.6.1 -->
<title>Contact | Samson Koelle</title>
<meta name="generator" content="Jekyll v3.6.3" />
<meta property="og:title" content="Contact" />
<meta name="author" content="Paul Le" />
<meta property="og:locale" content="en_US" />
<meta name="description" content="Powered by Jekyll and GitHub Pages" />
<meta property="og:description" content="Powered by Jekyll and GitHub Pages" />
<link rel="canonical" href="http://localhost:4000/pages/research.html" />
<meta property="og:url" content="http://localhost:4000/pages/research.html" />
<meta property="og:site_name" content="Samson Koelle" />
<script type="application/ld+json">
{"description":"Powered by Jekyll and GitHub Pages","headline":"Contact","@type":"WebPage","url":"http://localhost:4000/pages/research.html","author":{"@type":"Person","name":"Paul Le"},"@context":"https://schema.org"}</script>
<!-- End Jekyll SEO tag -->


</head>
<body>

<div class="site-container">

  <header>
  <div class="logo">
    <a href="/">Samson Koelle</a>
  </div>
  <nav>
    
      <a href="/pages/research.html">Research</a>
    
      <a href="/pages/news.html">Updates</a>
    
      <a href="/pages/about.html">About</a>
    
  </nav>
</header>


  <section>
    <h1 id="samson-koelle">Preprints</h1>
    <div class="page-content">
  <h1 id="samson-koelle">A regression approach for explaining manifold embedding coordinates </h1>
    <h2 id="samson-koelle">Meila M, Koelle S, Zhang H 
      (2018) (<a href="https://arxiv.org/abs/1811.11891">Link </a> )</h2>
<p> <img src="/assets/img/projects/arxivflasso.png" alt="" style="float:right;width:3in;height:auto;"> 
  Manifold embedding algorithms map high dimensional data, down to coordinates in a much lower dimensional space. One of the aims of the dimension reduction is to find the intrinsic coordinates that describe the data manifold. However, the coordinates returned by the embedding algorithm are abstract coordinates. Finding their physical, domain related meaning is not formalized and left to the domain experts. This paper studies the problem of recovering the domain-specific meaning of the new low dimensional representation in a semi-automatic, principled fashion. We propose a method to explain embedding coordinates on a manifold as non-linear compositions of functions from a user-defined dictionary. We show that this problem can be set up as a sparse linear Group Lasso recovery problem, find sufficient recovery conditions, and demonstrate its effectiveness on data.
</p>
</div>

<h1 id="samson-koelle">Publications</h1>
  <div class="page-content">
    <h1 id="samson-koelle">Statistical inference for partially observed branching processes with application to cell lineage tracking of in vivo hematopoiesis</h1>
    <h2 id="samson-koelle">Xu J, Koelle S, ... (2019) Annals of Applied Statistics
    (<a href="https://projecteuclid.org/euclid.aoas/1574910037">Link</a>) </h2>
<p> <img src="/assets/img/projects/stochastic.png" alt="" style="float:right;width:3in;height:auto;">
  Single-cell lineage tracking strategies enabled by recent experimental technologies have produced significant insights into cell fate decisions, but lack the quantitative framework necessary for rigorous statistical analysis of mechanistic models describing cell division and differentiation. In this paper, we develop such a framework with corresponding moment-based parameter estimation techniques for continuous-time, multi-type branching processes. Such processes provide a probabilistic model of how cells divide and differentiate, and we apply our method to study hematopoiesis, the mechanism of blood cell production. We derive closed-form expressions for higher moments in a general class of such models. These analytical results allow us to efficiently estimate parameters of much richer statistical models of hematopoiesis than those used in previous statistical studies. To our knowledge, the method provides the first rate inference procedure for fitting such models to time series data generated from cellular barcoding experiments. After validating the methodology in simulation studies, we apply our estimator to hematopoietic lineage tracking data from rhesus macaques. Our analysis provides a more complete understanding of cell fate decisions during hematopoiesis in nonhuman primates, which may be more relevant to human biology and clinical strategies than previous findings from murine studies. For example, in addition to previously estimated hematopoietic stem cell self-renewal rate, we are able to estimate fate decision probabilities and to compare structurally distinct models of hematopoiesis using cross validation. These estimates of fate decision probabilities and our model selection results should help biologists compare competing hypotheses about how progenitor cells differentiate. The methodology is transferrable to a large class of stochastic compartmental and multi-type branching models, commonly used in studies of cancer progression, epidemiology and many other fields.
</p>
</div>
  <div class="page-content">
    <h1 id="samson-koelle">Acquired cancer resistance to combination immunotherapy from transcriptional loss of class I HLA</h1>
    <h2 id="samson-koelle">Paulson K, ..., Koelle S, ... (2018) Nature Communications
    (<a href="https://www.nature.com/articles/s41467-018-06300-3">Link</a>) </h2>
<p> <img src="/assets/img/projects/tsne.png" alt="" style="float:right;width:3in;height:auto;">
 Understanding mechanisms of late/acquired cancer immunotherapy resistance is critical to improve outcomes; cellular immunotherapy trials offer a means to probe complex tumor–immune interfaces through defined T cell/antigen interactions. We treated two patients with metastatic Merkel cell carcinoma with autologous Merkel cell polyomavirus specific CD8+ T cells and immune-checkpoint inhibitors. In both cases, dramatic remissions were associated with dense infiltration of activated CD8+s into the regressing tumors. However, late relapses developed at 22 and 18 months, respectively. Here we report single cell RNA sequencing identified dynamic transcriptional suppression of the specific HLA genes presenting the targeted viral epitope in the resistant tumor as a consequence of intense CD8-mediated immunologic pressure; this is distinguished from genetic HLA-loss by its reversibility with drugs. Transcriptional suppression of Class I loci may underlie resistance to other immunotherapies, including checkpoint inhibitors, and have implications for the design of improved immunotherapy treatments. 
</p>

</div>

<div class="page-content">
    <h1 id="samson-koelle">Clonal expansion and compartmentalized maintenance of rhesus macaque NK cell subsets.</h1> 
    <h2 id="samson-koelle"> Wu C, ..., Koelle S, ... (2018) Science Immunology
    (<a href="https://immunology.sciencemag.org/content/3/29/eaat9781.long">Link</a>) </h2>
<p> <img src="/assets/img/projects/F4.large.png" alt="" style="float:right;width:3in;height:auto;">
  Natural killer (NK) cells recognize and eliminate infected and malignant cells. Their life histories are poorly understood, particularly in humans, due to lack of informative models and endogenous clonal markers. Here, we apply transplantation of barcoded rhesus macaque hematopoietic cells to interrogate the landscape of NK cell production, expansion, and life histories at a clonal level long term and after proliferative challenge. We identify oligoclonal populations of rhesus CD56−CD16+ NK cells that are characterized by marked expansions and contractions over time yet remained long-term clonally uncoupled from other hematopoietic lineages, including CD56+CD16− NK cells. Individual or groups of CD56−CD16+ expanded clones segregated with surface expression of specific killer immunoglobulin-like receptors. These clonally distinct NK cell subpopulation patterns persisted for more than 4 years, including after transient in vivo anti-CD16–mediated depletion and subsequent regeneration. Profound and sustained interleukin-15–mediated depletion was required to generate new oligoclonal CD56−CD16+ NK cells. Together, our results indicate that linear NK cell production from multipotent hematopoietic progenitors or less mature CD56+CD16− cells is negligible during homeostasis and moderate proliferative stress. In such settings, peripheral compartmentalized self-renewal can maintain the composition of distinct, differentiated NK cell subpopulations.
</p>
</div>

<div class="page-content">
    <h1 id="samson-koelle">Barcoding of Macaque Hematopoietic Stem and Progenitor Cells: A Robust Platform to Assess Vector Genotoxicity.</h1> 
    <h2 id="samson-koelle"> Yabe I, ..., Koelle S, ... (2018) Molecular Therapy Methods and Clinical Development
    (<a href="https://www.cell.com/molecular-therapy-family/methods/fulltext/S2329-0501(18)30105-0">Link</a>) </h2>
<p> <img src="/assets/img/projects/gr1_lrg.png" alt="" style="float:right;width:3in;height:auto;">
  Gene therapies using integrating retrovirus vectors to modify hematopoietic stem and progenitor cells have shown great promise for the treatment of immune system and hematologic diseases. However, activation of proto-oncogenes via insertional mutagenesis has resulted in the development of leukemia. We have utilized cellular bar coding to investigate the impact of different vector designs on the clonal behavior of hematopoietic stem and progenitor cells (HSPCs) during in vivo expansion, as a quantitative surrogate assay for genotoxicity in a non-human primate model with high relevance for human biology. We transplanted two rhesus macaques with autologous CD34+ HSPCs transduced with three lentiviral vectors containing different promoters and/or enhancers of a predicted range of genotoxicities, each containing a high-diversity barcode library that uniquely tags each individual transduced HSPC. Analysis of clonal output from thousands of individual HSPCs transduced with these barcoded vectors revealed sustained clonal diversity, with no progressive dominance of clones containing any of the three vectors for up to almost 3 years post-transplantation. Our data support a low genotoxic risk for lentivirus vectors in HSPCs, even those containing strong promoters and/or enhancers. Additionally, this flexible system can be used for the testing of future vector designs.
</p>
</div>

<div class="page-content">
    <h1 id="samson-koelle">Barcoding of Macaque Hematopoietic Stem and Progenitor Cells: A Robust Platform to Assess Vector Genotoxicity.</h1> 
    <h2 id="samson-koelle"> Yabe I, ..., Koelle S, ... (2018) Molecular Therapy Methods and Clinical Development
    (<a href="https://www.cell.com/molecular-therapy-family/methods/fulltext/S2329-0501(18)30105-0">Link</a>) </h2>
<p> <img src="/assets/img/projects/gr1_lrg.png" alt="" style="float:right;width:3in;height:auto;">
  Gene therapies using integrating retrovirus vectors to modify hematopoietic stem and progenitor cells have shown great promise for the treatment of immune system and hematologic diseases. However, activation of proto-oncogenes via insertional mutagenesis has resulted in the development of leukemia. We have utilized cellular bar coding to investigate the impact of different vector designs on the clonal behavior of hematopoietic stem and progenitor cells (HSPCs) during in vivo expansion, as a quantitative surrogate assay for genotoxicity in a non-human primate model with high relevance for human biology. We transplanted two rhesus macaques with autologous CD34+ HSPCs transduced with three lentiviral vectors containing different promoters and/or enhancers of a predicted range of genotoxicities, each containing a high-diversity barcode library that uniquely tags each individual transduced HSPC. Analysis of clonal output from thousands of individual HSPCs transduced with these barcoded vectors revealed sustained clonal diversity, with no progressive dominance of clones containing any of the three vectors for up to almost 3 years post-transplantation. Our data support a low genotoxic risk for lentivirus vectors in HSPCs, even those containing strong promoters and/or enhancers. Additionally, this flexible system can be used for the testing of future vector designs.
</p>
</div>

<div class="page-content">
    <h1 id="samson-koelle">The impact of aging on primate hematopoiesis as interrogated by clonal tracking.</h1> 
    <h2 id="samson-koelle"> Yu K, ..., Koelle S, ... (2018) Blood
    (<a href="https://www.ncbi.nlm.nih.gov/pubmed/29295845">Link</a>) </h2>
<p> <img src="/assets/img/projects/blood802033absf1.png" alt="" style="float:right;width:3in;height:auto;">
  Age-associated changes in hematopoietic stem and progenitor cells (HSPCs) have been carefully documented in mouse models but poorly characterized in primates and humans. To investigate clinically relevant aspects of hematopoietic aging, we compared the clonal output of thousands of genetically barcoded HSPCs in aged vs young macaques after autologous transplantation. Aged macaques showed delayed emergence of output from multipotent (MP) clones, with persistence of lineage-biased clones for many months after engraftment. In contrast to murine aging models reporting persistence of myeloid-biased HSPCs, aged macaques demonstrated persistent output from both B-cell and myeloid-biased clones. Clonal expansions of MP, myeloid-biased, and B-biased clones occurred in aged macaques, providing a potential model for human clonal hematopoiesis of indeterminate prognosis. These results suggest that long-term MP HSPC output is impaired in aged macaques, resulting in differences in the kinetics and lineage reconstitution patterns between young and aged primates in an autologous transplantation setting.
</p>
</div>

<div class="page-content">
    <h1 id="samson-koelle">Geographic Clonal Tracking in Macaques Provides Insights into HSPC Migration and Differentiation. </h1> 
    <h2 id="samson-koelle"> Wu C, ..., Koelle S, ... (2018) Journal of Experimental Medicine
    (<a href="https://www.ncbi.nlm.nih.gov/pubmed/29141868">Link</a>) </h2>
<p> <img src="/assets/img/projects/jem_20171341_fig2.png" alt="" style="float:right;width:3in;height:auto;">
  The geographic distribution of hematopoiesis at a clonal level is of interest in understanding how hematopoietic stem and progenitor cells (HSPCs) and their progeny interact with bone marrow (BM) niches during regeneration. We tagged rhesus macaque autologous HSPCs with genetic barcodes, allowing clonal tracking over time and space after transplantation. We found marked geographic segregation of CD34+ HSPCs for at least 6 mo posttransplantation, followed by very gradual clonal mixing at different BM sites over subsequent months to years. Clonal mapping was used to document local production of granulocytes, monocytes, B cells, and CD56+ natural killer (NK) cells. In contrast, CD16+CD56- NK cells were not produced in the BM, and in fact were clonally distinct from multipotent progenitors producing all other lineages. Most surprisingly, we documented local BM production of CD3+ T cells early after transplantation, using both clonal mapping and intravascular versus tissue-resident T cell staining, suggesting a thymus-independent T cell developmental pathway operating during BM regeneration, perhaps before thymic recovery.
</p>
</div>

  <div class="page-content">
    <h1 id="samson-koelle">Discussion of 'Quantitative stability of hematopoietic stem and progenitor cell clonal output in rhesus macaques receiving transplants'</h1> 
    <h2 id="samson-koelle"> Koelle S, ..., Dunbar CE (2017) Blood
    (<a href="https://www.ncbi.nlm.nih.gov/pubmed/28087539">Link</a>) </h2>
<p> <img src="/assets/img/projects/koelleblood.png" alt="" style="float:right;width:3in;height:auto;">
  In this issue of Blood, Koelle et al demonstrate that hematopoietic output originates from multiple multipotent clones that are stable over time, with little evidence of clonal succession. The hematopoietic compartment has the daunting task of daily producing an enormous quantity of various blood cells. This daily miracle relies upon a very limited number of super athlete cells called hematopoietic stem cells (HSCs). The HSC reigns on a hierarchical system where self-renewal capacity and differentiation toward specific lineages are finely regulated.  An in-depth understanding of hematopoiesis has obvious implications for leukemogenesis, stem cell transplantation, gene therapy, and biology of aging. Studying the clonal dynamics of hematopoiesis is challenging because the cells of interest are elusive and their progeny not readily identifiable. Fortunately, recent technological advances have supported the development of different models and strategies that provided new insights in the biology of HSCs.
</p>
</div>
  <div class="page-content">
    <h1 id="samson-koelle">Clonal tracking of rhesus macaque hematopoiesis highlights a distinct lineage origin for natural killer cells</h1>
    <h2 id="samson-koelle"> Wu C, ..., Koelle S, ... (2014) Cell Stem Cell
    (<a href="https://www.ncbi.nlm.nih.gov/pubmed/24702997">Link</a>) </h2>
    <p> <img src="/assets/img/projects/csc.png" alt="" style="float:right;width:3in;height:auto;">
<!--
<p> <img src="/assets/img/projects/proj-1/merkel.png" alt="" style="float:right;width:404px;height:404px;">
-->
<!-- <img src="/assets/img/projects/proj-1/merkel.png" style='height: 100%; width: 100%; object-fit: contain'/>
-->
Analysis of hematopoietic stem cell function in nonhuman primates provides insights that are relevant for human biology and therapeutic strategies. In this study, we applied quantitative genetic barcoding to track the clonal output of transplanted autologous rhesus macaque hematopoietic stem and progenitor cells over a time period of up to 9.5 months. We found that unilineage short-term progenitors reconstituted myeloid and lymphoid lineages at 1 month but were supplanted over time by multilineage clones, initially myeloid restricted, then myeloid-B clones, and then stable myeloid-B-T multilineage, long-term repopulating clones. Surprisingly, reconstitution of the natural killer (NK) cell lineage, and particularly the major CD16(+)/CD56(-) peripheral blood NK compartment, showed limited clonal overlap with T, B, or myeloid lineages, and therefore appears to be ontologically distinct. Thus, in addition to providing insights into clonal behavior over time, our analysis suggests an unexpected paradigm for the relationship between NK cells and other hematopoietic lineages in primates.
</p>
</div>

</section>


</div>

<script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.1/jquery.min.js"></script>
<script src="/assets/js/functions.js"></script>
</body>
</html>
